### PR TITLE
Bug 1219249 - Disable multi-tap on the keypad. r=thills

### PR DESCRIPTION
**Pull request by gabrielesvelto**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=3353838c0827f80c960069263d2fe2b708ef718a

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=d95152634a8a637d56533c539aa1c45c66968900

File: shared/js/dialer/keypad.js [hunk 1]
```diff
@@ -312,6 +312,14 @@ var KeypadManager = {
...
     this._lastPressedKey = key;
 
+    // If user input number more 50 digits, app shouldn't accept.
+    // The limit only applies while not on a call - there is no
+    // limit while on a call (bug 917630).
+    if (key !== 'delete' && !this._onCall &&
+        (this._phoneNumber.length >= this.kMaxDigits)) {
+      return;
+    }
+
```
Comment (tamarahills) [inline shared/js/dialer/keypad.js]: Do we need to make these changes on emergency-call keypad.js?  I was just thinking there could be some emergency related implications there, but haven't thought through all of them.

Comment (gabrielesvelto) [inline shared/js/dialer/keypad.js]: You're right, the emergency call app should suffer from the same issue.

**Pull request merged**